warp-core/src/main/scala/com/workday/warp/TestId.scala

```diff
@@ -41,5 +40,73 @@ trait TestId {
     * @return some fully qualified method name.
     */
   @throws[RuntimeException]
-  final def testId: String = this.maybeTestId.get
+  final def id: String = this.maybeId.get
```

Comment:
What are the cases where this might throw an exception?
Also, teeny nitpick: `maybeTestId` -> `maybeId` in the comment above

Reply:
good question, this would throw an exception if either `maybeTestClass` or `maybeTestMethod` were empty. i think that would happen when using dynamic tests but haven't tried it yet.

We can create a `TestId` with an `id`, but empty `maybeTestClass` and `maybeTestMethod` like this:

```scala
new TestId(Failure(new ClassNotFoundException), Failure(new NoSuchMethodException)) {
    override lazy val maybeId: Try[String] = Success("com.workday.warp.Test.foo")
  }
```